integrations/threejs/threejs.html

```diff
@@ -0,0 +1,295 @@
+		v_uv = position.xy * vec2(0.5, 0.5) + vec2(0.5, 0.5);
+        gl_Position = vec4(position.xyz, 1.0);
+      }
+    </script>
+    <script id="shader-fs" type="x-shader/x-fragment">
+      precision mediump float;
+      precision highp int;
+      
+      //Input Format
+      in highp vec2 v_uv;
+
+      uniform sampler2D u_texture;
+      uniform sampler2D u_depth;
+      
+      //layout(location = 0) out highp vec4 out_var_SV_Target0;
+
+      void main()
+      {
+        vec4 col = texture2D(u_texture, v_uv);
+        float depth = texture2D(u_depth, v_uv).x;
+        depth = (depth + 1.0) / 2.0;
+
+        gl_FragColor = vec4(col.zyx, 1.0);
+        gl_FragDepth = depth;
+      }
+    </script>
+  </head>
+  <body>
+    <script type="importmap">
+      {
+        "imports": {
+          "three": "./js/three.module.js",
+          "three/addons/": "./js/jsm/"
+        }
+      }
+
+    </script>
+
+    <script type="module">
+      import * as THREE from "three";
+
+      import Stats from "three/addons/libs/stats.module.js";
+      import WebGL from "three/addons/capabilities/WebGL.js";
+      import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+      if (WebGL.isWebGL2Available() === false) {
+        document.body.appendChild(WebGL.getWebGL2ErrorMessage());
+      }
+
+      THREE.Object3D.DEFAULT_UP.set(0, 0, 1);
+
+      let scene, mesh, renderer, stats;
+      let colourTextureGL, udTextureDepth;
+
+      camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.1, 1000 );
+      camera.position.y = -1;
+
+      scene = new THREE.Scene(); 
+
+      renderer = new THREE.WebGLRenderer();
+      renderer.setPixelRatio( window.devicePixelRatio );
+      renderer.setSize(window.innerWidth, window.innerHeight);
+      document.body.appendChild(renderer.domElement);
+
+      const controls = new OrbitControls( camera, renderer.domElement );
+      controls.update();
+
+      stats = new Stats();
+      document.body.appendChild(stats.dom);
+      window.addEventListener("resize", onWindowResize);
+
+      let material = new THREE.ShaderMaterial({
+        uniforms: {
+          u_texture: { type: "t", value: null },
+          u_depth: { type: "t", value: null },
+        },
+        vertexShader: document.getElementById("shader-vs").textContent.trim(),
+        fragmentShader: document.getElementById("shader-fs").textContent.trim(),
+      });
+
+      let geometry = new THREE.BufferGeometry();
+      const vertices = new Float32Array([-1,-1,0,  1,-1,0,  1,1,0, -1,-1,0,  1,1,0,  -1,1,0]);
+      geometry.setAttribute(
+        "position",
+        new THREE.BufferAttribute(vertices, 3)
+      );
+      mesh = new THREE.Mesh(geometry, material);
+      mesh.position.set( -0.5, -0.5, 0 );
+      scene.add(mesh);
+
+      ///// test
+      const boxGeometry = new THREE.BoxGeometry( 0.25,  0.25,  0.25 );
+      const texture = new THREE.TextureLoader().load( 'crate.gif' );
```

Comment:
I would change the box to untextured. Then you don't need to add 'crate.gif' into the sample.